src/app/page.tsx

```diff
@@ -1,78 +1,100 @@
 "use client";
-import { useState } from "react";
+import React, { useState } from "react";
+import Navbar from "@/components/Navbar";
+import "./styles.css";
+import { initializeApp } from 'firebase/app';
+import { getFirestore, collection, addDoc } from 'firebase/firestore';
+import firebaseConfig from '../../firebase'; 
+import app from '../../firebase';
+import db from '../../firebase';
 
-import Footer from "@/components/Footer";
-import Login from "@/components/Login";
-import Link from "next/link";
 
-export default function Home() {
-  const [openModal, setOpenModal] = useState(false);
+const Home: React.FC = () => {

```

Comment:
keep the function structure the way it was originally: `export default function pageOrComponentName ({props}: pageOrComponentNameProps) `